example/static/less/styles.less

```diff
@@ -3,139 +3,107 @@
  * Source: https://github.com/caktus/django-scribbler
  * Docs: http://django-scribbler.readthedocs.org/
  *
- * Copyright 2012, Caktus Consulting Group, LLC
+ * Copyright 2012-6, Caktus Consulting Group, LLC
  * BSD License
  *
-*/
-
-/* Bootstrap imports
-================================================== */
-@import "bootstrap/reset.less";
-@import "bootstrap/variables.less";
-@import "bootstrap/mixins.less";
-@import "bootstrap/scaffolding.less";
-@import "bootstrap/grid.less";
-@import "bootstrap/layouts.less";
-@import "bootstrap/type.less";
-@import "bootstrap/code.less";
-@import "bootstrap/forms.less";
-@import "bootstrap/tables.less";
-@import "bootstrap/buttons.less";
-@import "bootstrap/alerts.less";
-@import "bootstrap/close.less";
-@import "bootstrap/navs.less";
-@import "bootstrap/navbar.less";
-@import "bootstrap/dropdowns.less";
-@import "bootstrap/carousel.less";
-@import "bootstrap/hero-unit.less";
-@import "bootstrap/responsive.less";
-@import "bootstrap/utilities.less";
-@import "bootstrap/tooltip.less";
-@import "bootstrap/sprites.less";
-@import "bootstrap/breadcrumbs.less";
-@import "bootstrap/component-animations.less";
-@import "bootstrap/wells.less";
-@import "bootstrap/labels-badges.less";
-@import "bootstrap/pagination.less";
+ */
 
 /* Variables
 ======================== */
-@baseFontFamily: @serifFontFamily;
 @scribblerRed: #CD0000;
+@black: #000;
+@white: #fff;
+@gray: #555;
+@grayLighter: #eee;
 
 /* Generic styles
 ======================== */
-body { 
-	margin: 0;
-	background: @grayLighter; 
-	&.login-home {
-		#gradient .radial;
-		background-repeat: repeat;
-	}
+body {
+    margin: 0;
+    background: @grayLighter;
+    &.login-home {
+        background: @gray;
+    }
 }
 
 @font-face {
     font-family: 'cutiveregular';
     src: url('../fonts/cutive-regular-webfont.eot');
     src: url('../fonts/cutive-regular-webfont.eot?#iefix') format('embedded-opentype'),
-         url('../fonts/cutive-regular-webfont.woff') format('woff'),
-         url('../fonts/cutive-regular-webfont.ttf') format('truetype'),
-         url('../fonts/cutive-regular-webfont.svg#cutiveregular') format('svg');
+    url('../fonts/cutive-regular-webfont.woff') format('woff'),
+    url('../fonts/cutive-regular-webfont.ttf') format('truetype'),
+    url('../fonts/cutive-regular-webfont.svg#cutiveregular') format('svg');
```

Comment:
I think it'd be clearer if these three lines were left indented as before; they appear to go with the `src:` key on line 32.